test/helm/image_registry_test.go

```
@@ -0,0 +1,150 @@
+			t.Errorf("per-image registry override not honored: operator image %q; want myreg.io/seaweedfs-operator:<tag>", operator)
+		}
+	})
+}
+
+// renderImages runs `helm template` with the given extra args and returns
+// a map of container name -> image for every container and init container
+// across all rendered Deployments and Jobs.
+func renderImages(t *testing.T, chartDir string, extraArgs ...string) map[string]string {
+	t.Helper()
+	args := append([]string{"template", "image-test", chartDir}, extraArgs...)
+	cmd := exec.Command("helm", args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)